packages/foreman/rubygem-hocon/rubygem-hocon.spec

```
@@ -0,0 +1,95 @@
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name hocon
+%global gem_require_name %{gem_name}
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.3.1
+Release: 1%{?dist}
+Summary: HOCON Config Library
+Group: Development/Languages
+License: Apache License, v2
```

This isn't a valid license code
```suggestion
License: ASL 2.0
```